components/Header/index.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
```

Comment:
Please keep a format, you are calling the imports with double quotes but on other files it is just single quote. I would rather have one quote and no ;